src/statistics/DataChartBox.cpp

```diff
@@ -129,19 +129,11 @@ void DataChartBox::update(
 
 qreal DataChartBox::axisYMax() const
 {
-    if (axisYMax_ == Y_MAX_DEFAULT)
```

Comment:
I removed this thinking the constants won't change in the future, otherwise it should stay.